In [1]:
import os
import glob
import pandas as pd

In [12]:
page_xml_path = (
    r"\\ad\collections\TwoCenturies\TwoCenturies IV\Incunabula\*column pages Transkribus export"
    r"\*\*_column_model_output\page\*.xml"
)

In [13]:
xmls = glob.glob(page_xml_path)

In [14]:
len(xmls)

2278

In [19]:
sizes = pd.DataFrame([(x, os.stat(x).st_size) for x in xmls], columns=['Path', 'Filesize'])

In [24]:
sizes.query("2000 < Filesize < 5000").head().values

array([['\\\\ad\\collections\\TwoCenturies\\TwoCenturies IV\\Incunabula\\BMC_1 4 column pages Transkribus export\\1282976\\BMC_1_4_column_model_output\\page\\0023_J_2704_aa_30_1_0136.xml',
        2315],
       ['\\\\ad\\collections\\TwoCenturies\\TwoCenturies IV\\Incunabula\\BMC_2 4 column pages Transkribus export\\1268109\\BMC_2_4_column_model_output\\page\\0047_J_2704_aa_30_2_0114.xml',
        3830],
       ['\\\\ad\\collections\\TwoCenturies\\TwoCenturies IV\\Incunabula\\BMC_2 4 column pages Transkribus export\\1268109\\BMC_2_4_column_model_output\\page\\0065_J_2704_aa_30_2_0199.xml',
        3335],
       ['\\\\ad\\collections\\TwoCenturies\\TwoCenturies IV\\Incunabula\\BMC_5 4 column pages Transkribus export\\1268132\\BMC_5_4_column_model_output\\page\\0170_J_2704_aa_30_5_0453.xml',
        2549],
       ['\\\\ad\\collections\\TwoCenturies\\TwoCenturies IV\\Incunabula\\BMC_7 4 column pages Transkribus export\\1268145\\BMC_7_4_column_model_output\\page\\0078_J_2704_aa_30_7_0200.x

Using this one for unit tests?  
"\\\\ad\\collections\\TwoCenturies\\TwoCenturies IV\\Incunabula\\BMC_7 4 column pages Transkribus export\\1268145\\BMC_7_4_column_model_output\\page\\0078_J_2704_aa_30_7_0200.xml"